### PR TITLE
I corrected some spelling

### PR DESCRIPTION
**Pull request by goku2610**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ echo "'use strict'; function hello() { print('Hello World'); } hello();" | ./bin
 
 ## Contributing
 
-The main purpose of this repository is to continue to evolve Hermes, making it faster and more efficient. We are grateful to the community for contributing bugfixes and improvements. Read below to learn how you can take part in improving Hermes.
+The main purpose of this repository is to continue to evolve Hermes, making it faster and more efficient. We are grateful to the community for contributing bug fixes and improvements. Read below to learn how you can take part in improving Hermes.
```
Comment (vitkarpov) [inline README.md]: I should be bugfix (or bug-fix), shouldn't it? (https://www.quora.com/Is-the-word-bugfix-or-bug-fix)

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/facebook/hermes/blob/master/CONTRIBUTING.md)
 <img src="./website/static/img/logo.svg" alt="Hermes logo - large H with wings" align="right" width="20%"/>
 
-Hermes is a JavaScript engine optimized for fast start up of [React Native](https://facebook.github.io/react-native/) apps on Android. It features ahead-of-time static optimization and compact bytecode.
+Hermes is a JavaScript engine optimized for the fast start-up of [React Native](https://facebook.github.io/react-native/) apps on Android. It features ahead-of-time static optimization and compact bytecode.
```
Comment (vitkarpov) [inline README.md]: It's a merge conflict with #124 
A funny fact: there's exactly the same change for "startup" https://github.com/facebook/hermes/pull/124#discussion_r332102268

Comment (rickhanlonii) [inline README.md:7]:
```suggestion
Hermes is a JavaScript engine optimized for fast start up of [React Native](https://facebook.github.io/react-native/) apps on Android. It features ahead-of-time static optimization and compact bytecode.
```

File: README.md (hunk 1 above)
Comment (rickhanlonii) [inline README.md:50]:
```suggestion
The main purpose of this repository is to continue to evolve Hermes, making it faster and more efficient. We are grateful to the community for contributing bug-fixes and improvements. Read below to learn how you can take part in improving Hermes.
```

**Comment by rickhanlonii** [comment]
@goku2610 I fixed up your PR but we need you to sign the CLA in order to import it.

If you want to sign that then I'll re-open and land the typo fix 👍

**Pull request closed without merging**